File: swagger/models/get_class_visits_request.py
from .base_model import BaseModel


class GetClassVisitsRequest(BaseModel):
	"""
	no description available

	Attributes:

		class_id
		The class ID.

		last_modified_date
		When included in the request, only records modified on or after the
		`LastModifiedDate` specified are included in the response.
	"""

	input_map = {
		'ClassID': 'class_id',
		'LastModifiedDate': 'last_modified_date',
		}

	output_map = {
		'class_id': 'ClassID',
		'last_modified_date': 'LastModifiedDate',
		}

File: swagger/Models/GetClassVisitsRequest.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class GetClassVisitsRequest extends BaseModel
{
	/**
	 * The class ID.
	 * 
	 * @var integer
	 */
	public $classId;
	/**
	 * When included in the request, only records modified on or after the
	 * `LastModifiedDate` specified are included in the response.
	 * 
	 * @var string
	 */
	public $lastModifiedDate;

	protected function getInputMap()
	{
		return [
			'ClassID' => 'classId',
			'LastModifiedDate' => 'lastModifiedDate',
			];
	}

	protected function getOutputMap()
	{
		return [
			'classId' => 'ClassID',
			'lastModifiedDate' => 'LastModifiedDate',
			];
	}
}
